server/router.go

```diff
@@ -30,6 +31,28 @@ func HandleError(s *client.Schemas, t func(http.ResponseWriter, *http.Request) e
 	}))
 }
 
+func proxyProtoHandler(h http.Handler) http.Handler {
+	fn := func(w http.ResponseWriter, r *http.Request) {
+		requestUrl:=r.Header.Get(api.DEFAULT_OVERRIDE_URL_HEADER)
```

Comment:
I think it will cause `go fmt` problem because `:=` should have space in both side.